woocommerce/woocommerce-android
# [Simplified login] initial screen for Jeptack installation

Closes: #7847 


### Description
This PR implements the initial screen of native Jetpack installation/connection flow.
Even though the PR seems big, it's just because it adds a new screen with its boilerplate.

ℹ️ I'm still waiting on the designer's confirmation on the usage of the illustrations for dark mode, but I don't think this should block the PR, we can always update them separately.

⚠️ Please don't merge before #7795 

### Testing instructions
1. Make sure you force the usage of the native Jetpack installation (either by adding your device to the list of test devices in the AB test in Firebase Console, or by hardcoding the value in `JetpackInstallationExperiment`)
2. Open the app, then sign in using your WordPress.com account.
3. In the site picker, click on Add Store, then choose "Connect an existing store".
4. Continue the steps of the following test cases:

###### Jetpack not installed flow
5. Enter the address of a site that doesn't have Jetpack.
6. Click on Continue
7. Confirm that the screen matching the Jetpack installation is shown.

##### Jetpack installed
5. Enter the address of a site that has Jetpack.
6. Click on Continue.
7. Confirm that the screen matching the Jetpack connection is shown.

### Images/gif
| | light | dark |
| --- | --- | --- |
| Install | ![image](https://user-images.githubusercontent.com/1657201/201871369-5d10024e-1b5e-4fc8-af14-62cd504988d6.png) | ![image](https://user-images.githubusercontent.com/1657201/201871231-5986ecb3-931a-4e83-a420-0f4116f1a091.png) |
| Connect | ![image](https://user-images.githubusercontent.com/1657201/201871710-abb941e8-4acc-456f-a1ca-503def8f9231.png) | ![image](https://user-images.githubusercontent.com/1657201/201871795-9a58762b-1cc3-4adc-9d86-4b83ecd3770b.png) |
